src/routes/home/components/GameListItem.vue

```diff
@@ -106,11 +98,16 @@ export default {
   },
   computed: {
     ...mapStores(useGameStore, useAuthStore, useGameListStore),
-    numPlayersReady() {
-      return this.p0ready + this.p1ready;
-    },
-    readyText() {
-      return `${this.numPlayers} / 2`;
+    playersText() {
+      const numPlayers = this.players.length;
+      if (numPlayers === 0) {
+        return 'Empty';
+      } else if (numPlayers === 1) {
+        return `vs ${this.players[0].username}`;
+      } else if (numPlayers === 2) {
+        return `${this.players[0].username} vs ${this.players[1].username}`;
+      }
+      return '';
```

Comment:
This looks right, but stylistically we generally use switch statements for multiple conditions on the same variable